model/src/main/kotlin/OrtResult.kt

```diff
@@ -146,8 +146,8 @@ data class OrtResult(
      * depth of [maxLevel] where counting starts at 0 (for the [Project] or [Package] itself) and 1 are direct
      * dependencies etc. A value below 0 means to not limit the depth.
      */
-    fun collectDependencies(id: Identifier, maxLevel: Int = -1): SortedSet<Identifier> {
-        val dependencies = sortedSetOf<Identifier>()
+    fun collectDependencies(id: Identifier, maxLevel: Int = -1): Set<Identifier> {
```

Comment:
Are you sure this doesn't have any side effects? I believe we're using these directly when creating the result files, so I would assume that sorting *does* matter.

Reply:
I will double check it again. I first wanted to see if there is any test complaining about any change in the ordering.

Reply:
This looks like a test breakage for exactly that reason: https://ci.appveyor.com/project/heremaps/oss-review-toolkit/builds/30860989/tests

Reply:
I checked again the call hierarchy and I could not find any assumption about the ordering which would lead to issues, e.g. change of order in one of the reports.